src/tarantool.client/Converters/PacketSizeConverter.cs

```diff
@@ -19,8 +20,31 @@ public void Write(PacketSize value, IMsgPackWriter writer)
         {
             writer.Write(DataTypes.UInt32);
 
-            var requestIdBytes = BitConverter.GetBytes(value.Value).Reverse().ToArray();
-            writer.Write(requestIdBytes);
+            var binary = new UIntBinary(value.Value);
+
+            byte[] bytes;
```

Comment:
This trick requires next round with check if Buffer on the level of writer is going to improve performance - issue in github is going to be enough

Overall solution needs to be exposed to the rest of similar conversion methods in msgpack - this might be low hanging fruit